### PR TITLE
feat: Add a stub text display class (#5804)

### PR DESCRIPTION
**Pull request by kaidokert**
Closes https://github.com/shaka-project/shaka-player/issues/5804

### CODE REVIEW

File: lib/player.js [hunk 1]
```diff
@@ -5445,7 +5446,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         return new shaka.text.UITextDisplayer(
             this.video_, this.videoContainer_);
       } else {
-        return new shaka.text.SimpleTextDisplayer(this.video_);
+        if ('addTextTrack' in this.video_) {
```
Comment (kaidokert) [inline lib/player.js:5449]: Unsure about whether this is the cleanest and most appropriate way to do a feature check here.

Comment (absidue) [reply lib/player.js:5449]: Probably also worth logging a warning pointing out that text tracks won't work, because the browser doesn't support them.

Comment (kaidokert) [reply lib/player.js:5449]: Good call, added a line.

File: lib/player.js [hunk 2]
```diff
@@ -5445,7 +5446,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
...
```
Comment (avelad) [inline lib/player.js:5449]: Can we use `HTMLMediaElement.prototype.addTextTrack`  instead?

Comment (kaidokert) [reply lib/player.js]: Done - thanks

**Review by avelad — APPROVED** [review]

File: lib/player.js (hunk 2 above)
Comment (kaidokert) [reply lib/player.js]: Doh, i just saw that this requires a lint disable, `// eslint-disable-next-line no-restricted-syntax` ?

Comment (avelad) [reply lib/player.js]: Yes, It's necessary

Comment (kaidokert) [reply lib/player.js]: ack thanks again

**Pull request merged**